backend/scripts/db_snapshot.sql

```diff
@@ -0,0 +1,333 @@
+-- Generated via: pg_dump -Fp --no-privileges --no-owner --no-comments --schema-only 
```

Comment:
This is an empty db schema based on the structure in the current dev environment, plus one row from the alembic migrations table.